### PR TITLE
feat: remove hidden tables toggle

### PR DESCRIPTION
**Pull request by owlas**
Closes: #3322 

### Description:

Removes the option to show all tables in the UI

### CODE REVIEW

**Comment by hamzahc1** [comment]
Do you think there should be a prompt on the explore view (similar to sql runner) saying "missing tables?" that links them to the tables config page in the settings?

File: packages/backend/src/routers/projectRouter.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@ projectRouter.get('/explores', isAuthenticated, async (req, res, next) => {
             await projectService.getAllExploresSummary(
                 req.user!,
                 req.params.projectUuid,
-                req.query.filtered === 'true',
```
Comment (ZeRego) [inline packages/backend/src/routers/projectRouter.ts:94]: This is going to break the project table configuration page (ProjectTablesConfiguration) where it expects to get all the tables.

**Review by ZeRego — CHANGES_REQUESTED** [review]
ps: You deleted the events.yml file by mistake.

**Comment by owlas** [comment]
I don't think that prompt will be relevant very often so I wouldn't want to add more things to the explorer page, where pixels are at a premium!

**Comment by ZeRego** [comment]
Closing in favour of #3357

**Pull request closed without merging**